dangerfile.ts

```diff
@@ -0,0 +1,40 @@
+import { danger } from 'danger';
+import * as logger from '@markbind/core/src/utils/logger';
+
+interface Couplings {
+  [key: string]: string[];
+}
+
+const couplings: Couplings = {
+  '.github/workflows/check-file-diff.yml': [
+    'package.json',
+    'babel.config.js',
+  ],
+  'babel.config.js': [
+    'package.json',
+    'lerna.json',
+  ],
+};
```

Comment:
Could we add more legitimate examples here!

Reply:
Alright, then I will remove the testing couplings completely.